using-valtimo/forms/configure-object-form.md

```diff
@@ -0,0 +1,21 @@
+# Configuring an Objecten API object form
+
+In order to display objects from the Objecten API a form can be used to define the way the data is shown. This form
+will be used when opening the details of an object from the 
+[case detail tabs for an object type](/extending-valtimo/tabs/configure-tab-object-type.md).
+
+Which form is used to display an object is determined based on the name of the form. The name of the form needs to be
+the name of the objecttype follow by the `.editform` suffix. For example, to create a form to display objects of type
```

Comment:
_The name of the form needs to be the name of the objecttype `followed` by the .editform suffix._
just a suggestion

Reply:
fixed